Production/govuk_ios/Services/AppConfigService.swift

```diff
@@ -32,24 +32,26 @@ public final class AppConfigService: AppConfigServiceInterface {
             filename: ConfigStrings.filename.rawValue,
             completion: { [weak self] result in
                 guard let self = self else { return }
-                try? self.handleResult(result)
+                self.handleResult(result)
             }
         )
 
         appConfigServiceClient.fetchAppConfig(
             completion: { [weak self] result in
                 guard let self = self else { return }
-                try? self.handleResult(result)
+                self.handleResult(result)
             }
         )
     }
 
-    private func handleResult(_ result: Result<AppConfig, AppConfigError>) throws {
+    private func handleResult(_ result: Result<AppConfig, AppConfigError>) {
         switch result {
         case .success(let appConfig):
             setConfig(appConfig.config)
         case .failure(let error):
-            throw error
+            if error == .invalidSignatureError {
```

Comment:
What happens if `remoteJsonError` is thrown? I would probably just remove this if statement and set `isAppAvailable` to false regardless of the error.

Reply:
I did initially do the same as what your suggestion says but then I thought the ticket is very specific to invalidSignature which is why I added the if statement etc.

I have followed your comment now and changed to set the app to unavailable for all failures.